packages/banner/package.json

```diff
@@ -32,6 +32,7 @@
   },
   "devDependencies": {
     "@pluralsight/ps-design-system-build": "^1.10.1",
+    "@pluralsight/ps-design-system-button": "*",
```

Comment:
sad, it would have been great to get this in css.  I read your comment and feel the pain on why.  This is another reason esm needs to get here.